src/utils/searchUtils/searchUtils.ts

```diff
@@ -140,11 +140,7 @@ export function getAntsRequiringUpdate({
     .map(([id, ant]) => {
       // if user is not the owner, skip
       if (!ant.state?.Owner || ant?.state.Owner !== userAddress) return;
-      if (
-        !AntHandlerNames.filter((h) => h !== 'evolve').every((handler) =>
-          ant.handlers?.includes(handler),
-        )
-      )
+      if (!AntHandlerNames.every((handler) => ant.handlers?.includes(handler)))
```

Comment:
Seems like the filter was removed here but that it was put in intentionally in the previous PR. Is this change back intentional?